manifests/0000_90_cluster-image-registry-operator_02_operator-servicemonitor.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: image-registry-operator
+  namespace: openshift-image-registry
+spec:
+  endpoints:
+  - targetPort: 60000
+    path: /metrics
+    scheme: http
```

Comment:
i realize this is a WIP so it might be on your radar already but just a heads up that this will need to be switched to https before merging.

Reply:
Thanks for the heads up @bparees . I am cutting corners to have everything in place, later on I have to revisit everything.

Reply:
Done, metrics are now served over https.